### ISSUE TITLE
Using Revisionable with morphMaps doesn't work

### ISSUE THREAD

**Issue by djave-co** [VentureCraft/revisionable#318]
When I use this package as stated in the docs, my morphMaps are ignored. I have the following in my `AppServiceProvider`:

    Relation::morphMap([
        'users' => 'App\Artixy\Models\User',
        'arts' => 'App\Artixy\Models\Art',
    ]);

This means that in the database, revisionable saves off values like so:

```
233
**users**
19
19
telephone
NULL
123234345
2018-05-31 18:32:01
2018-05-31 18:32:01
```

Note that it is saved as **users**, not App\Models\User. How can I amend this? It creates the error:

`Class 'users' not found`

**Comment by jayzyaj** [comment]
What laravel are you using. It didn't work on Laravel 5.5 for me

**Comment by duellsy** [comment]
closed in #326
